content/posts/newsletter/2020-03-15.md

```diff
@@ -47,14 +50,20 @@ After the Prosody team [released Snikket](https://blog.prosody.im/introducing-sn
 
 Sam Whited wrote about the [Go XMPP library](https://blog.samwhited.com/2020/02/extensions-in-mellium/).
 
+Turns out [Grindr uses #XMPP for their in-app chat](https://mastodon.xyz/@jcbrand/103757814572529323)
+
+emus created a blog post from his brainstorming and talk on a vision for XMPP he created during the Berlin "Online" Sprint.
```

Comment:
What's the policy regarding handles and real names?

Above you wrote Sam Whited, and here you write `emus`. When I wrote the newsletter, I wrote the real names (and IIRC usually gave the handle as well).

Reply:
Still awaits my action :see_no_evil:

Reply:
@jcbrand please launch the rocket! :rocket: